packages/suite/src/views/wallet/send/components/TotalSent.tsx

```diff
@@ -7,8 +7,7 @@ import { Card, Translation, FiatValue, FormattedCryptoAmount } from 'src/compone
 const StyledCard = styled(Card)`
     display: flex;
     flex-direction: row;
-    justify-items: space-between;
-    align-items: center;
+    place-items: center space-between;
```

Comment:
@tomasklim This is not working at least on my machine:
<img width="837" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/0baca61f-2978-4170-8478-d20785916126">